### ISSUE TITLE
Please include support for "$web" blob container

### ISSUE THREAD

**Issue by gmantri** [Azure/azure-storage-node#450]
Recently at Build, support for static website hosting is announced. Static website contents go into a special blob container called "$web". However this name is not supported by the SDK (only "$root" and "$logs" names are supported). Can we please change the SDK to support "$web" blob container?

I believe we need to change `containerNameIsValid` method in `validate.js` file. The modified code should be something like this:

```
/**
* Validates a container name.
*
* @param {string} containerName The container name.
*/
exports.containerNameIsValid = function (containerName, callback) {
  var fail;
  initCallback(callback, function (f, cb) {
    fail = f;
    callback = cb;
  });

  var nameError = getNameError(containerName, 'Container');

  if (!nameError || containerName.match(/^(\$root|\$logs|\$web)$/)) {
    callback();
    return true;
  } else {
    return fail(nameError);
  }
};

```

**Comment by XiaoningLiu** [comment]
@gmantri That's correct. We already have this change planned in next release.

**Comment by gmantri** [comment]
Thank you for your prompt response. Much appreciated. Is there an ETA on the next release?

**Comment by XiaoningLiu** [comment]
A release is planned in next week, we are waiting for some internal testing and validation for other new features included in this release.

**Comment by gmantri** [comment]
Thanks. Good to know about the timeline. Let's keep this issue open until then if it is all right with you.

**Comment by XiaoningLiu** [comment]
Update: we will release a patch version for the security issue reported  in #449, and next minor version for $web and other features.

**Comment by XiaoningLiu** [comment]
$web support is released in 2.9.0-preview.

**Comment by gmantri** [comment]
I just tried it and it works perfectly fine. Thank you team for quick turnaround on this issue.
